### PR TITLE
rec: add copy files for *-from-file configuration directives

### PR DESCRIPTION
**Pull request by SrX**

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -48,8 +48,33 @@
...
     owner: "{{ pdns_rec_file_owner }}"
     group: "{{ pdns_rec_file_group }}"
-    mode: 0750
+    mode: "{{ pdns_rec_config_include_dir_mode }}"
   when: "pdns_rec_config['include-dir'] is defined"
+  register: _pdns_recursor_configuration_include_dir
+
+- block:
+
+  - name: Ensure that the PowerDNS Recursor configuration from-files directory exists
```
Comment (npmdnl) [inline tasks/configure.yml:57]: wrong indentation

**Review by npmdnl — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -182,6 +182,23 @@ pdns_rec_service_overrides:
...
+```yaml
+pdns_rec_config_from_files_dir_mode: 0750
+pdns_rec_config_from_files: []
+#pdns_rec_config_from_files:
+#  - dest: "/var/lib/pdns-recursor/from-files/forward-zones.txt"
+#    src: "files/forward-zones/forward.txt"
+```
+
+List of files to copy to the PowerDNS Recursor instance, could be used for the `*-from-file` settings in the `recursor.conf` configuration file.
+The `pdns_rec_config_from_files_dir_mode` allow to change the mode of files if required.
```
Comment (npmdnl) [inline README.md:194]: allows to change

File: tasks/configure.yml [hunk 3]
```diff
@@ -48,8 +48,33 @@
     state: directory
     owner: "{{ pdns_rec_file_owner }}"
     group: "{{ pdns_rec_file_group }}"
-    mode: 0750
+    mode: "{{ pdns_rec_config_include_dir_mode }}"
   when: "pdns_rec_config['include-dir'] is defined"
+  register: _pdns_recursor_configuration_include_dir
```
[the remark this answers is not in the document]
Comment (SrX) [reply tasks/configure.yml:53]: Register attribute is on the same level as the task, I think it's ok, isn't?

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#registering-variables

File: tasks/configure.yml (hunk 1 above)
Comment (SrX) [reply tasks/configure.yml:57]: The block indentation require some extra indentation of tasks inside it. 
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_blocks.html#grouping-tasks-with-blocks

File: README.md (hunk 2 above)
Comment (SrX) [reply README.md:194]: fixed.

**Review by npmdnl — APPROVED** [review]
LGTM

**Pull request merged**